test-requirements.txt

```diff
@@ -25,3 +25,4 @@ psutil==5.4.5
 webtest==2.0.25
 rstcheck>=3.3.0,<3.4
 pyrabbit
+python-keyczar
```

Comment:
StackStorm doesn't depend on `python-keyczar` anymore, but we still needs it for tests where we check our code and output is fully compatible with keyczar one.

Reply:
Yep, will do.

I thought I already added it, but probably to the other file.